draft-ietf-masque-connect-ip.md

```diff
@@ -1316,7 +1331,9 @@ of hardware offloads.
 
 When the protocol running inside the tunnel uses congestion control (e.g.,
 {{TCP}} or {{QUIC}}), the proxied traffic will incur at least two nested
-congestion controllers. When tunneled packets are sent using QUIC DATAGRAM
+congestion controllers. Nested congestion controlers can lead to
+pathological behavior, such as in the case of "TCP meltdown"
```

Comment:
I was thinking we can reference this as the worst case, since we could run CONNECT-IP on H2 proxies with end to end TCP flows, so sometimes you will have TCP in TCP.

I don't know a better reference — happy to swap if someone has one.

Reply:
Is it behind a paywall? I was able to download <https://www.spiedigitallibrary.org/proceedings/Download?urlId=10.1117%2F12.630496> directly, and wasn't presented any login wall.

Reply:
> I was thinking we can reference this as the worst case, since we could run CONNECT-IP on H2 proxies with end to end TCP flows, so sometimes you will have TCP in TCP.

But in that case it should be in the next paragraph, because that's the one that tackles TCP-in-TCP.

> I don't know a better reference — happy to swap if someone has one.

I don't think anyone has ever researched nested congestion control without nested loss recovery (nor nested flow control). Thinking about this more I think that the QUIC DATAGRAM frame is the first protocol to have nested congestion control without loss recovery nor flow control. That would explain why there isn't any research on this yet.

> Is it behind a paywall? I was able to download https://www.spiedigitallibrary.org/proceedings/Download?urlId=10.1117%2F12.630496 directly, and wasn't presented any login wall.

Could it be that our employers have access so you can download this from the office but I can't from home? Can you try from your phone on cellular?

Base on all this, I'd say maybe let's just not add this reference?

Reply:
You're right, I couldn't access this from my home network without a paywall.